libs/ledger-live-common/src/families/elrond/js-synchronisation.ts

```diff
@@ -9,10 +9,29 @@ import {
 } from "./api";
 import elrondBuildESDTTokenAccounts from "./js-buildSubAccounts";
 import { reconciliateSubAccounts } from "./js-reconciliation";
-import { FEES_BALANCE } from "./constants";
-import { TokenAccount } from "@ledgerhq/types-live";
+import { Operation, SubAccount, TokenAccount } from "@ledgerhq/types-live";
 import { computeDelegationBalance } from "./logic";
-import BigNumber from "bignumber.js";
+
+function pruneOperations(
```

Comment:
Could you please explain why there is a need to prune operations? My concern is to understand why we have duplicate operations in the first place.
Didn't have time to look into that area, but I was suspecting more the `reconciliateSubAccounts` part, did you look in that direction?

Reply:
Got it, ok if the duplication originates from the backend itself then frontend has to deduplicate, makes sense. Thanks.